### PR TITLE
color: add Equals() method to compare two colors

### PR DESCRIPTION
**Pull request by andrewaustin**

### CODE REVIEW

**Comment by andrewaustin** [comment]
Hey fatih,

We talked about this a few weeks ago on slack and I finally got around to implementing it. Let me know if it needs any more work to get accepted.

File: color.go [hunk 1]
```diff
@@ -259,6 +259,31 @@ func (c *Color) isNoColorSet() bool {
...
+
+	for _, attr := range c.params {
+		if !c2.attrExists(attr) {
+			return false
+		}
+	}
+
+	return true
+
+}
```
Comment (fatih) [inline color.go]: Can you remove the whitespace here?

**Comment by fatih** [comment]
Hi @andrewaustin. LGTM, this is cool! I've made a simple comment, after that I think this is safe to merge :)

**Comment by andrewaustin** [comment]
Oops, removed that extra white space.

**Comment by fatih** [comment]
Thanks @andrewaustin :+1:

**Pull request merged**